Modules/Registration/Common/test/itkPointsLocatorTest.cxx

```diff
@@ -81,12 +81,21 @@ testPointsLocatorTest()
     return EXIT_FAILURE;
   }
 
-  std::cout << "Test:  Search() 1" << std::endl;
+  std::cout << "Test:  FindClosestNPoints() with distances  (1)" << std::endl;
+  std::vector<double> distances;
+  pointsLocator->FindClosestNPoints(coords, 10u, neighborhood, distances);
+  if (neighborhood.size() != 10 || distances.size() != 10)
+  {
+    std::cerr << "Error with FindClosestNPoints()" << std::endl;
```

Comment:
Although this is always expected to pass, please add an informative message according to the ITK SW Guide:
https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/blob/master/SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex#L3583

It is useful to have at least that information if the method fails in case someone adds a new test.